src/shared.ts

```diff
@@ -30,6 +32,8 @@ export const componentNameMapping: Record<
   "account-management": "stripe-connect-account-management",
   "notification-banner": "stripe-connect-notification-banner",
   "instant-payouts": "stripe-connect-instant-payouts",
+  "issuing-card": "stripe-connect-issuing-card",
```

Comment:
Note: support for setter methods in the SDK is still being added, so you'll have to come back here and add those later!